### PR TITLE
PCSSWebService

### PR DESCRIPTION
**Pull request by KFloodBCGov**
Moved all the SOAP ports to a single service

### CODE REVIEW

File: src/pcsswebservice/pom.xml [hunk 1]
```diff
@@ -99,7 +99,11 @@
 							<sourceRoot>${project.build.directory}/generated-sources/cxf</sourceRoot>
 							<wsdlOptions>
 								<wsdlOption>
+									<wsdl>${basedir}/src/main/resources/wsdl/pcssCivil.wsdl</wsdl>
```
Comment (akroon3r) [inline src/pcsswebservice/pom.xml:102]: BASE_DIR for an Env var?

Comment (akroon3r) [reply src/pcsswebservice/pom.xml:102]: Not sure if standards for java are different for that...just asking what the norm for this is.

File: src/pcsswebservice/src/main/resources/wsdl/JusticeCDDS.wsdl [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        <soap12:body parts="parameters" use="literal"/>
+      </wsdl:output>
+    </wsdl:operation>
+  </wsdl:binding>
+  <wsdl:service name="JusticeCDDS.wsProvider.cdds">
+    <wsdl:port name="JusticeCDDS_wsProvider_cdds_Port" binding="tns:JusticeCDDS_wsProvider_cdds_Binder">
+      <soap12:address location="http://reeks.bcgov:5556/ws/JusticeCDDS.wsProvider.cdds/JusticeCDDS_wsProvider_cdds_Port"/>
+    </wsdl:port>
+  </wsdl:service>
+</wsdl:definitions>
```
Comment (akroon3r) [inline src/pcsswebservice/src/main/resources/wsdl/JusticeCDDS.wsdl:135]: <3

File: src/pcsswebservice/pom.xml (hunk 1 above)
Comment (KFloodBCGov) [reply src/pcsswebservice/pom.xml:102]: This is a maven var I think, same as ${project.basedir} ?

Comment (akroon3r) [reply src/pcsswebservice/pom.xml:102]: sounds good

**Comment by akroon3r** [comment]
I'll let Alex be the official sign off.

File: src/pcsswebservice/pom.xml (hunk 1 above)
Comment (alexjoybc) [reply src/pcsswebservice/pom.xml:102]: it is ;-)

File: src/pcsswebservice/src/main/resources/wsdl/JusticeCDDS.wsdl (hunk 2 above)
Comment (alexjoybc) [reply src/pcsswebservice/src/main/resources/wsdl/JusticeCDDS.wsdl:135]: We are keeping this to be compatible with webMethods sucks but that's what it is...

Comment (akroon3r) [reply src/pcsswebservice/src/main/resources/wsdl/JusticeCDDS.wsdl:135]: I was mentioning the lack of missing newlines.

File: src/pcsswebservice/pom.xml [hunk 3]
```diff
@@ -9,10 +9,10 @@
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>ca.bc.gov.open.pcss</groupId>
-	<artifactId>pcss-criminal-api</artifactId>
+	<artifactId>pcsswebservice</artifactId>
```
Comment (alexjoybc) [inline src/pcsswebservice/pom.xml:12]: can we name it pcss-web-service ? you just have to change the artifact id and the folder name

File: src/pcsswebservice/src/main/java/ca/bc/gov/open/pcss/pcsswebservice/WebServiceConfig.java [hunk 4]
```diff
@@ -31,31 +30,55 @@ public DispatcherServletPath dispatcherServletPathProvider() {
...
+    }
 
     @Bean
-    public LoggingFeature loggingFeature() {
+    public Endpoint CommonEndpoint(Bus bus, PcssCommonEndpoint pcssCommonEndpoint) {
 
-        LoggingFeature loggingFeature = new LoggingFeature();
-        loggingFeature.setPrettyLogging(true);
+        EndpointImpl endpoint = new EndpointImpl(bus, pcssCommonEndpoint);
+        endpoint.publish("/pcssCommon");
```
Comment (alexjoybc) [inline src/pcsswebservice/src/main/java/ca/bc/gov/open/pcss/pcsswebservice/WebServiceConfig.java:52]: Can we get the name from `webMethods: pacakge.whatever:serviceWs`

File: src/pcsswebservice/pom.xml (hunk 3 above)
Comment (KFloodBCGov) [reply src/pcsswebservice/pom.xml]: Done!

File: src/pcsswebservice/src/main/java/ca/bc/gov/open/pcss/pcsswebservice/WebServiceConfig.java (hunk 4 above)
Comment (KFloodBCGov) [reply src/pcsswebservice/src/main/java/ca/bc/gov/open/pcss/pcsswebservice/WebServiceConfig.java]: And done!

**Pull request merged**